cypress/integration/ampSpec.js

```diff
@@ -49,6 +49,14 @@ describe('AMP Tests on a .amp page', () => {
       .should('be', 2); // 1 for amp.js + 1 that Cypress injects into the head
   });
 
+  it('should contain an amp-img', () => {
+    const figure = getElement('figure').eq(0);
+    figure.should('be.visible');
+    figure.within(() => {
+      getElement('amp-img');
```

Comment:
Hmmm does this make an assertion? `getElement('amp-img')` could return null but as we're not checking its value, the tests would pass.

```suggestion
      getElement('amp-img').should('be.visible');
```